samples/idea-latest.yaml

```diff
@@ -0,0 +1,27 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: idea-latest
+spec:
+  started: true
+  template:
+    projects:
+      - name: web-nodejs-sample
```

Comment:
The project name should probably be changed to something like `python-hello-world`

Reply:
Done